packages/client/css/glsp-sprotty.css

```diff
@@ -50,23 +51,59 @@
 
 .sprotty-resize-handle {
     /* radius does not work in Firefox */
-    r: 7px;
+    r: 5px;
     fill: #884;
     stroke: none;
     z-index: 1000;
 }
 
 .sprotty-resize-handle.selected {
-    fill: #66a;
+    fill: rgb(41, 43, 46);
 }
 
 .sprotty-resize-handle.active {
-    fill: #0074e8;
+    fill: rgb(41, 43, 46);
 }
 
-.sprotty-resize-handle.mouseover {
-    stroke: #112;
-    stroke-width: 1;
+.sprotty-edge {
+    fill: none;
+    stroke-width: 1.5px;
+}
+
+.sprotty-edge.selected {
+    stroke-width: 1.5px;
+}
+
+.sprotty-edge.mouseover:not(.selected) {
+    stroke-width: 1.5px;
+}
+
+.sprotty-node:not(.selected) {
+    stroke-width: 0px;
+}
+
+.sprotty-node.mouseover:not(.selected),
+.sprotty-edge.mouseover:not(.selected) {
+    stroke: #1d1e1f;
+    stroke-width: 1.5px;
+    stroke-dashoffset: 5;
+    stroke-dasharray: 5, 5;
```

Comment:
In general, I like the visual differentiation for selection/mouseover via dashed stroke. But due to the dashed line, edges might flicker on mouseover (see also screenshot). Maybe we could think of a solid line in a contrast color?

![glsp-example-dashed-flickering](https://user-images.githubusercontent.com/47414588/148770442-e313cd3f-1d5a-49b2-b4a5-25bb5531846f.gif)

Reply:
Thanks Nina, you are right! We typically fix those issues by adding a transparent wider edge above the visible edge, which helps selection the edge too.
However, we also want to stay very generic, so probably dashes aren't a good choice for hover feedback.
I replaced it with opacity for now.
What do you think?
Also feel free to suggest different colors for the nodes if you don't like them.